src/json-patch-duplex.ts

```diff
@@ -677,4 +678,8 @@ if (typeof exports !== "undefined") {
   exports.validate = jsonpatch.validate;
   exports.validator = jsonpatch.validator;
   exports.JsonPatchError = jsonpatch.JsonPatchError;
-}
+
+  /* TS Transpiler automatically adds
+  .default  when referencing */
+  exports.default = jsonpatch;
```

Comment:
after https://github.com/Starcounter-Jack/JSON-Patch/pull/137#discussion_r93655435

yet another `if(typeof exports` is not needed, I thought about shortening even more.